keras_cv/layers/regularization/drop_path.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class DropPath(tf.keras.layers.Layer):
+    """
+    Implements the DropPath layer. Some samples from the batch are randomly
```

Comment:
what are "Samples" here?  Are they entire training samples along axis 0?

Let's also use DropPath as the actor in the sentence structure:

DropPath randomly drops samples during training with a probability of `rate`.

Reply:
@LukeWood I think it would also make sense to add a sentence noting the difference between DropPath and StochasticDepth?

Reply:
Great idea

Reply:
> Are they entire training samples along axis 0?

It drops some samples along the axis 0. I'll add the following sentence: "DropPath randomly drops some of the individual samples from a batch, whereas StachasticDepth randomly drops the entire batch."

Reply:
Awesome!  Resolve this conversation when done.